test/shared/utils.py

```diff
@@ -37,10 +50,17 @@ def cibuildwheel_run(project_path, env=None, add_env=None):
     if add_env is not None:
         env.update(add_env)
 
-    subprocess.check_call(
-        [sys.executable, '-m', 'cibuildwheel', project_path],
-        env=env,
-    )
+    with TemporaryDirectory() as output_dir:
+        try:
+            subprocess.check_call(
+                [sys.executable, '-m', 'cibuildwheel', '--output-dir', output_dir, project_path],
```

Comment:
Other annoying detail: in #179 (about which @joerick is not fully conviced yet, either, as far as I understood), @Czaki needs `subprocess.run` rather than `subprocess.check_call`. Then again, he already wasn't using `utils.cibuildwheel_run`, so maybe that's not relevant.

Reply:
But now this test (from #179) has symilar structure to other. After this changes, even after refactor to new style it will have "different structure"